guides.md

```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
 9. Visit [the corresponding cloud inbox](https://www.mailinator.com/v3/index.jsp?zone=public&query=hyphacoop) and confirm your message has arrived, It might take a few minutes.
 10. Don’t forget to email Infra WG as mentioned in onboarding email.
 
+## Expense Reimbursement
+
+Employees should try to submit eligible expenses in the same quarter they are incurred.
+The Finance WG reviews submitted expenses before each pay period, and pays out reimbursement amounts on the next pay day.
+
+### Submitting an expense
+
+1. Upload the expense receipt to [Employee Expenses](https://link.hypha.coop/receipts) with file name `xx-YYYY-MM-DD.ext` where `xx` is the initials of the member and `ext` the file extension, then make note of the URL.
+
+1. Open the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and fill in a new row according to instructions in the sheet.
+    - Date of Expense: `[date on your receipt]`
+    - Employee: `[employee to reimburse]`
+    - Description of Expense: `[short description of what the expense is for]`
+    - Total Amount: `[total receipt amount, including HST]`
+    - HST Amount: `[total HST on the receipt]` 
+    - Currency: `[currency of the receipt]` (default to `CAD`)
+    - Exchange Rate: `[currency exchange rate to CAD]` (default to `1.000` for CAD)
+    - Receipt Link: `[link to uploaded receipt]`
+
+1. That's it.
+    Once finance approves the expense it will show up beside your name on the [Employee Payroll](http://link.hypha.coop/payroll) sheet and you will be reimbursed in that pay period.
+    The amount will _not_ appear on your tax forms.
+
+### Approving an expense for reimbursement
+
+For the Finance WG to approve an expense:
+
+1. Verify all expenses that do not have a `Pay Period of Reimbursement` selected.
+
+1. Select a `Cost Center / Project` and `Pay Period of Reimbursement`.
+
+1. Verify that the `Total Amount (CAD)` is added to the [Reimbursement Summary](http://link.hypha.coop/reimbursement) sheet to the correct `Pay Period of Reimbursement` and `Employee`.
+
+1. Verify that the amount shows up correctly on the `Expense Reimbursement` column on [Employee Payroll](http://link.hypha.coop/payroll) of the applicable pay period, so it gets entered into Wagepoint on the next payroll run.
+
+1. 🚧 Track the expense appropriately in [Quickbooks Online](https://quickbooks.intuit.com/ca/).
```

Comment:
What does the bollard mean here?

Reply:
This part is not yet well defined :) I think after first run, when we work with bookkeeper to match accounts, we'll add this.